api/src/main/java/org/pmiops/workbench/rdr/RdrTaskQueue.java

```diff
@@ -0,0 +1,92 @@
+package org.pmiops.workbench.rdr;
```

Comment:
This functionality is taken from the [OfflineRdrExportController ](https://github.com/all-of-us/workbench/blob/master/api/src/main/java/org/pmiops/workbench/api/OfflineRdrExportController.java)
I will remove the methods from that file and have it use these